packages/flutter/lib/src/material/material_button.dart

```diff
@@ -415,6 +415,7 @@ class MaterialButton extends StatelessWidget {
       animationDuration: buttonTheme.getAnimationDuration(this),
       child: child,
       materialTapTargetSize: materialTapTargetSize ?? theme.materialTapTargetSize,
+      disabledElevation: disabledElevation,
```

Comment:
You could instead do this:
```suggestion
      disabledElevation: disabledElevation ?? 0.0,
```

Reply:
Right